content/numpy/concepts/ndarray/terms/transpose/transpose.md

```diff
@@ -52,3 +52,13 @@ This produces the following output:
  [2 5]
  [3 6]]
 ```
+
```

Comment:
Please provide a description of what the below example does.

Reply:
@Radhika-okhade done

thanks